examples/navigation/README.md

```diff
@@ -137,4 +137,34 @@ There are multiple content pages used as the content container. Namely `Home pag
 
 One of the content container types is `Listing items`, this type allows to select content type(s) that is/are then used to determine what is about to be queried in the page. To model it in Kontent [Type selector custom element](https://github.com/Simply007/kontent-custom-element-type-selector) is used and then in the application. Then when the application is registering the page components ir loads detail items and [register them under their own route](https://github.com/Simply007/Simply007.github.io/blob/source/gatsby-node.js#L115) with [detail component template](https://github.com/Simply007/Simply007.github.io/blob/source/src/templates/journal-item.js).
 
+## Set up Preview URLs
+
+If you are using complex and nested menu navigation in combination with [Preview URLs](https://docs.kontent.ai/tutorials/develop-apps/build-strong-foundation/set-up-preview#a-set-up-content-preview-in-your-project) you might need to have a way to map "preview" URL to the one you have build using schema customization, because in the context of Preview URLs there is only `url slug`, `codename`, `language` and `item ID`, but not information of other elements, or their relationship.
```

Comment:
```suggestion
If you are using complex and nested menu navigation in combination with [Preview URLs](https://docs.kontent.ai/tutorials/develop-apps/build-strong-foundation/set-up-preview#a-set-up-content-preview-in-your-project) you might need to have a way to map the "preview" URL to the one you have built using schema customization because in the context of Preview URLs there is only `url slug`, `codename`, `language` and `item ID`, but not the information of other elements, or their relationship.
```